tests/baseline_test.py

```diff
@@ -3,5 +3,5 @@
 def test_always_passes():
     assert True
 
-# def test_always_fails():
-#     assert False
+def test_always_fails():
```

Comment:
This change will cause the CI/CD workflow to _always_ fail, and cannot be accepted as-is.  There are valid reasons to use such, but additional code that uses `assert` to allow for false failures would also be required and is not represented in this commit.